### DISCUSSION TITLE
What I Talk About When I Talk About Query Optimizer (Part 1): IR Design

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#423] [Announcements]
# What I Talk About When I Talk About Query Optimizer (Part 1): IR Design

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2024/02-what-i-talk-about-when-i-talk-about-query-optimizer-part-1/

**Comment by slaiyer** [comment]
This article is such an excellent contribution towards keeping the world of software engineering open. Thanks, and subscribed!

**Comment by mohandotyes** [comment]
Excellent article its nice thank you so much for writing this wonderful content.
